process_pr.py

```diff
@@ -1732,6 +1756,8 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         requiresTestMessage = " (tests are also fine)"
     elif "tests-rejected" in labels:
         requiresTestMessage = " (but tests are reportedly failing)"
+    elif labels.intersection(set("tests-" + x for x in TEST_IGNORE_REASON)):
```

Comment:
it should be `set(labels)` here ... right?

Reply:
by the way , this condition will not hit as above we have `if "tests-approved" in labels:`. May be change it to 

```
if labels.intersection(set("tests-" + x for x in TEST_IGNORE_REASON)):
  requiresTestMessage = " (test failures were overridden)"
elif "tests-approved" in labels:
...
...
```

Reply:
> it should be `set(labels)` here ... right?

 `labels` is already `set` here  https://github.com/cms-sw/cms-bot/blob/ff41a6536c621196cca6769e02559dfc09885d46/process_pr.py#L1607